### PR TITLE
Added New Packages : Neon, Ncspot, Libmusicbrainz, Duckdb, xpup, frei0r-plugins, libcuefile, gavl, libreplaygain, libfaac, libfaad, libdca, liba52, liboil, schrodinger, libvdpau, libxv, libdv

### PR DESCRIPTION
**Pull request by tathastu871**

### CODE REVIEW

**Comment by tathastu871** [comment]
Please someone review and start checks

File: packages/duckdb/build.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+TERMUX_PKG_HOMEPAGE=https://www.duckdb.org
+TERMUX_PKG_DESCRIPTION="DuckDB is an in-process SQL OLAP Database Management System"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_VERSION=0.3.1
+TERMUX_PKG_SRCURL=https://github.com/duckdb/duckdb/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=ae2367d0a393be59e137ffa975f48f60b113b2a72aacc24fbc59afa0cbc3a511
+termux_step_pre_configure() {
+	LDFLAGS=" -llog"
```
Comment (buttaface) [inline packages/duckdb/build.sh:9]: Add the `+` I pointed out before.

File: packages/libfaad/build.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_LICENSE=GPL-2.0
+TERMUX_PKG_LICENSE_FILE="COPYING"
+TERMUX_PKG_VERSION=2.8.6
+_VERSION_REAL=$(echo $(cut -d '.' -f1,2 <<< ${TERMUX_PKG_VERSION}).0)
+TERMUX_PKG_SRCURL=https://sourceforge.net/projects/faac/files/faad2-src/faad2-${_VERSION_REAL}/faad2-${TERMUX_PKG_VERSION}.tar.gz/download
+TERMUX_PKG_SHA256=654977adbf62eb81f4fca00152aca58ce3b6dd157181b9edd7bed687a7c73f21
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="--build=aarch64-unknown-linux-gnu"
+termux_step_pre_configure() {
+	LDFLAGS=" -lm"
```
Comment (buttaface) [inline packages/libfaad/build.sh:13]: Same here.

File: packages/liboil/build.sh [hunk 3]
```diff
@@ -0,0 +1,10 @@
+TERMUX_PKG_HOMEPAGE=https://liboil.freedesktop.org/wiki/
+TERMUX_PKG_DESCRIPTION="Liboil is a library of simple functions that are optimized for various CPUs"
+TERMUX_PKG_LICENSE=custom
+TERMUX_PKG_LICENSE_FILE=COPYING
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=0.3.17
+TERMUX_PKG_SRCURL=https://liboil.freedesktop.org/download/liboil-${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=105f02079b0b50034c759db34b473ecb5704ffa20a5486b60a8b7698128bfc69
+TERMUX_PKG_DEPENDS="gtk-doc"
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="--build=aarch64-unknown-linux-gnu"
```
Comment (buttaface) [inline packages/liboil/build.sh:10]: This isn't going to work for other architectures.

File: packages/gavl/build.sh [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+TERMUX_PKG_DESCRIPTION="Gmerlin Audio Video Library"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_LICENSE=GPL-3.0
+TERMUX_PKG_LICENSE_FILE=COPYING
+TERMUX_PKG_VERSION=1.4.0
+TERMUX_PKG_SRCURL=https://sourceforge.net/projects/gmerlin/files/gavl/${TERMUX_PKG_VERSION}/gavl-${TERMUX_PKG_VERSION}.tar.gz/download
+TERMUX_PKG_DEPENDS="doxygen, libpng"
+TERMUX_PKG_SHA256=51aaac41391a915bd9bad07710957424b046410a276e7deaff24a870929d33ce
+termux_step_pre_configure() {
+	${TERMUX_PKG_SRCDIR}/autogen.sh
```
Comment (xeffyr) [inline packages/gavl/build.sh:11]: `./autogen.sh`

File: packages/liba52/build.sh [hunk 5]
```diff
@@ -0,0 +1,14 @@
+TERMUX_PKG_HOMEPAGE=https://liba52.sourceforge.io
+TERMUX_PKG_DESCRIPTION="library for decoding ATSC A/52 streams."
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_LICENSE=GPL-2.0
+TERMUX_PKG_LICENSE_FILE=COPYING
+TERMUX_PKG_VERSION=0.7.4
+TERMUX_PKG_SRCURL=https://liba52.sourceforge.io/files/a52dec-${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=a21d724ab3b3933330194353687df82c475b5dfb997513eef4c25de6c865ec33
+TERMUX_PKG_DEPENDS="ndk-sysroot"
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="--build=aarch64-unknown-linux-gnu"
```
Comment (xeffyr) [inline packages/liba52/build.sh:10]: This shouldn't be needed.

File: packages/libdca/build.sh [hunk 6]
```diff
@@ -0,0 +1,13 @@
+TERMUX_PKG_HOMEPAGE=https://www.videolan.org/developers/libdca.html
+TERMUX_PKG_DESCRIPTION="libdca is a free library for decoding DTS"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_LICENSE="COPYING"
+TERMUX_PKG_VERSION=0.0.7
+TERMUX_PKG_SRCURL=https://code.videolan.org/videolan/libdca/-/archive/${TERMUX_PKG_VERSION}/libdca-${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=758a025db36356d5df829d2204f7aca618c7dfbe1e5d98ab633df33a6e32d61a
+TERMUX_PKG_DEPENDS="ndk-sysroot"
+termux_step_pre_configure() {
+	cd ${TERMUX_PKG_SRCDIR} &&
```
Comment (xeffyr) [inline packages/libdca/build.sh:10]: No need to cd into srcdir. Don't use `&&` to split commands.

File: packages/libdca/build.sh [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+		sed '/sys\/soundcard.h/ s/sys/linux/' libao/audio_out_oss.c -i
+	${TERMUX_PKG_SRCDIR}/bootstrap
```
Comment (xeffyr) [inline packages/libdca/build.sh:12]: `./bootstrap`

File: packages/libdv/build.sh [hunk 8]
```diff
@@ -0,0 +1,10 @@
+TERMUX_PKG_HOMEPAGE=http://libdv.sourceforge.net/
+TERMUX_PKG_DESCRIPTION="Lib for Quasar DV Codec digital format"
+TERMUX_PKG_LICENSE=GPL-2.1
+TERMUX_PKG_LICENSE_FILE=COPYING
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=1.0.0
+TERMUX_PKG_SRCURL=http://ftp.debian.org/debian/pool/main/libd/libdv/libdv_${TERMUX_PKG_VERSION}.orig.tar.gz
+TERMUX_PKG_SHA256=a305734033a9c25541a59e8dd1c254409953269ea7c710c39e540bd8853389ba
+TERMUX_PKG_DEPENDS="libxv"
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="--build=aarch64-unknown-linux-gnu"
```
Comment (xeffyr) [inline packages/libdv/build.sh:10]: `--build` shouldn't be necessary, remove it. It is already configured by `build-package.sh`.

File: packages/libfaac/build.sh [hunk 9]
```diff
@@ -0,0 +1,10 @@
+TERMUX_PKG_HOMEPAGE=http://faac.sourceforge.net/
+TERMUX_PKG_DESCRIPTION="Freeware Advanced Audio Coder AAC Enoder"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_LICENSE=GPL-2.0
+TERMUX_PKG_LICENSE_FILE="COPYING"
+TERMUX_PKG_VERSION=1.29.9.2
+_VERSION_REAL=$(cut -f1,2 -d '.' <<< ${TERMUX_PKG_VERSION})
```
Comment (xeffyr) [inline packages/libfaac/build.sh:7]: Use slices. The cut command is overkill there.

Don't copy code snippets unless you understand why they needed. https://github.com/termux/termux-packages/pull/8174#discussion_r766960647

File: packages/libfaad/build.sh [hunk 10]
```diff
@@ -0,0 +1,14 @@
+
+TERMUX_PKG_HOMEPAGE=http://faac.sourceforge.net/
+TERMUX_PKG_DESCRIPTION="Freeware Advanced Audio Coder AAC Decoder"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_LICENSE=GPL-2.0
+TERMUX_PKG_LICENSE_FILE="COPYING"
+TERMUX_PKG_VERSION=2.8.6
+_VERSION_REAL=$(echo $(cut -d '.' -f1,2 <<< ${TERMUX_PKG_VERSION}).0)
```
Comment (xeffyr) [inline packages/libfaad/build.sh:8]: See previous comment about `cut` command. But here more issues - useless `echo`. Anyway, use slices here.

File: packages/libfaad/build.sh [hunk 11]
```diff
@@ -0,0 +1,14 @@
...
+TERMUX_PKG_SRCURL=https://sourceforge.net/projects/faac/files/faad2-src/faad2-${_VERSION_REAL}/faad2-${TERMUX_PKG_VERSION}.tar.gz/download
```
Comment (xeffyr) [inline packages/libfaad/build.sh:9]: Remove `/download`.

File: packages/libfaad/build.sh [hunk 12]
```diff
@@ -0,0 +1,14 @@
...
+TERMUX_PKG_SHA256=654977adbf62eb81f4fca00152aca58ce3b6dd157181b9edd7bed687a7c73f21
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="--build=aarch64-unknown-linux-gnu"
```
Comment (xeffyr) [inline packages/libfaad/build.sh:11]: `--build` shouldn't be necessary. See previous comments.

File: packages/libmusicbrainz/build.sh [hunk 13]
```diff
@@ -0,0 +1,9 @@
+TERMUX_PKG_HOMEPAGE=https://musicbrainz.org/
+TERMUX_PKG_DESCRIPTION="The MusicBrainz Client Library, also known as mb_client."
+TERMUX_PKG_LICENSE="LGPL-2.1"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=5.1.0
+TERMUX_PKG_SRCURL=https://github.com/metabrainz/libmusicbrainz/releases/download/release-${TERMUX_PKG_VERSION}/libmusicbrainz-5.1.0.tar.gz
```
Comment (xeffyr) [inline packages/libmusicbrainz/build.sh:6]: Don't hardcode version in URL.

File: packages/libreplaygain/build.sh [hunk 14]
```diff
@@ -0,0 +1,10 @@
+TERMUX_PKG_HOMEPAGE=https://www.musepack.net/
+TERMUX_PKG_DESCRIPTION="Library to implement ReplayGain standard for audio"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_LICENSE=LGPL
+TERMUX_PKG_VERSION=475
+TERMUX_PKG_SRCURL=https://files.musepack.net/source/libreplaygain_r${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=8258bf785547ac2cda43bb195e07522f0a3682f55abe97753c974609ec232482
+termux_step_pre_configure() { 
+	cd ${TERMUX_PKG_SRCDIR} && autoreconf -vif
```
Comment (xeffyr) [inline packages/libreplaygain/build.sh:9]: Just do `autoreconf -vif`.

File: packages/libvdpau/build.sh [hunk 15]
```diff
@@ -0,0 +1,9 @@
+TERMUX_PKG_HOMEPAGE=https://www.freedesktop.org/wiki/Software/VDPAU/
+TERMUX_PKG_DESCRIPTION="VDPAU is the Video Decode and Presentation API for UNIX"
+TERMUX_PKG_LICENSE=custom
+TERMUX_PKG_LICENSE_FILE=COPYING
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=1.4
+TERMUX_PKG_SRCURL=https://gitlab.freedesktop.org/vdpau/libvdpau/-/archive/${TERMUX_PKG_VERSION}/libvdpau-${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=240dd8f9ec08de707529917677827ba12e13b9fc299eeb3af13ea05e7fc74aba
+TERMUX_PKG_DEPENDS="x11*, xorgproto, libxext, ninja"
```
Comment (xeffyr) [inline packages/libvdpau/build.sh:9]: `x11*` - such package doesn't exit. If you meant wildcard there - never use that. It is not supported in dependency field and is wrong anyway.

File: packages/libxv/build.sh [hunk 16]
```diff
@@ -0,0 +1,12 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/freedesktop/xorg-lib-libXv
+TERMUX_PKG_DESCRIPTION="Xvideo extension library"
+TERMUX_PKG_LICENSE=MIT
+TERMUX_PKG_LICENSE_FILE=COPYING
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=1.0.11
+TERMUX_PKG_SRCURL=https://github.com/freedesktop/xorg-lib-libXv/archive/refs/tags/libXv-${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=7d608436370d85230a176aca25aa27d98861ac49c59e88ca4e2fb6a6759bedf1
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="--build=aarch64-unknown-linux-gnu"
```
Comment (xeffyr) [inline packages/libxv/build.sh:9]: Remove `--build` - see previous comments.

File: packages/libxv/build.sh [hunk 17]
```diff
@@ -0,0 +1,12 @@
...
+termux_step_pre_configure() {
+	${TERMUX_PKG_SRDIR}/autogen.sh
```
Comment (xeffyr) [inline packages/libxv/build.sh:11]: `./autogen.sh`

File: packages/ncspot/build.sh [hunk 18]
```diff
@@ -0,0 +1,15 @@
...
+TERMUX_PKG_LICENSE="BSD 2-Clause"
+TERMUX_PKG_VERSION=0.9.3
+TERMUX_PKG_SRCURL=https://github.com/hrkfdn/ncspot/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=ed873d007ce356e8c6eed56226533b686682a98d2a37487668416539d4e10e78
+TERMUX_INSTALL_DEPS=true
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_DEPENDS=" pulseaudio, x11-repo, python, rust, pulseaudio, dbus, ncurses, libxcb, openssl"
+termux_step_make_install() {
+	termux_setup_rust
+	cd ${TERMUX_PKG_SRCDIR} && cargo build --jobs $TERMUX_MAKE_PROCESSES --target $CARGO_TARGET_NAME --release
```
Comment (xeffyr) [inline packages/ncspot/build.sh:13]: No need to cd into srcdir. Use `TERMUX_PKG_BUILD_IN_SRC=true` instead specifically for this package.

File: packages/neon/build.sh [hunk 19]
```diff
@@ -0,0 +1,16 @@
...
+TERMUX_PKG_VERSION=0.32.1
+TERMUX_PKG_SRCURL=https://notroj.github.io/neon/neon-${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=05c54bc115030c89e463a4fb28d3a3f8215879528ba5ca70d676d3d21bf3af52
+TERMUX_PKG_DEPENDS="libxml2"
+TERMUX_PKG_SUGGESTS="libproxy, pakchois"
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="--prefix=$PREFIX --with-ssl --enable-shared"
+termux_step_pre_configure() {
+	#This step is needed if xmlto is not available
+	cd ${TERMUX_PKG_SRCDIR} && sed 's/\(install-\(html\|man\):\).*/\1/' -i Makefile.in
```
Comment (xeffyr) [inline packages/neon/build.sh:15]: Remove cd command.

File: packages/neon/build.sh [hunk 20]
```diff
@@ -0,0 +1,16 @@
...
+TERMUX_PKG_LICENSE=GPL-2.0
+TERMUX_PKG_LICENSE_FILE="src/COPYING.LIB"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=0.32.1
+TERMUX_PKG_SRCURL=https://notroj.github.io/neon/neon-${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=05c54bc115030c89e463a4fb28d3a3f8215879528ba5ca70d676d3d21bf3af52
+TERMUX_PKG_DEPENDS="libxml2"
+TERMUX_PKG_SUGGESTS="libproxy, pakchois"
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="--prefix=$PREFIX --with-ssl --enable-shared"
```
Comment (xeffyr) [inline packages/neon/build.sh:12]: Remove prefix. It is already configured by build system.

File: packages/schrodinger/build.sh [hunk 21]
```diff
@@ -0,0 +1,10 @@
+TERMUX_PKG_HOMEPAGE=http://schrodinger.sourceforge.net/
+TERMUX_PKG_DESCRIPTION="Portable implementation of Dirac"
+TERMUX_PKG_LICENSE="GPL-2.0, LGPL-2.0, MPL-2.0, MIT"
+TERMUX_PKG_LICENSE_FILE="COPYING.GPL, COPYING.LGPL,
```
Comment (xeffyr) [inline packages/schrodinger/build.sh:4]: Typo there.

File: packages/schrodinger/build.sh [hunk 22]
```diff
@@ -0,0 +1,10 @@
+TERMUX_PKG_HOMEPAGE=http://schrodinger.sourceforge.net/
+TERMUX_PKG_DESCRIPTION="Portable implementation of Dirac"
+TERMUX_PKG_LICENSE="GPL-2.0, LGPL-2.0, MPL-2.0, MIT"
+TERMUX_PKG_LICENSE_FILE="COPYING.GPL, COPYING.LGPL, 
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=1.0.0
+TERMUX_PKG_SRCURL=https://sourceforge.net/projects/schrodinger/files/schrodinger/${TERMUX_PKG_VERSION}/schroedinger-${TERMUX_PKG_VERSION}.tar.gz/download
+TERMUX_PKG_SHA256=1decf7c5f54eafa3944e8ca4e3fd34cb8ae4a648
+TERMUX_PKG_DEPENDS="x11-repo, gtk-doc, liboil"
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="--build=aarch64-unknown-linux-gnu"
```
Comment (xeffyr) [inline packages/schrodinger/build.sh:10]: Remove `--build`.

File: packages/xpup/build.sh [hunk 23]
```diff
@@ -0,0 +1,26 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/ericchiang/xpup
+TERMUX_PKG_DESCRIPTION="command line tool for processing XML"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=0.0.0+20150709172731.8b964504de6f-1
```
Comment (xeffyr) [inline packages/xpup/build.sh:5]: This is not valid version. If you don't know what to put there, use commit date.

**Pull request closed without merging**